tv-shows-app/src/components/features/shows/ShowListSection/ShowListSection.tsx

```diff
@@ -0,0 +1,39 @@
+import { Grid } from "@chakra-ui/react";
+import { useState } from "react";
+import useSWR from "swr";
+import { ShowCard } from "../ShowCard/ShowCard";
+import { fetcherSecure } from "@/fetchers/fetcher";
+import { Loader } from "@/components/shared/Loader/Loader";
+import { IShow } from "@/typings/Show.type";
+
+export const ShowListSection = ({ showList }) => {
+	const [shows, setShows] = useState([] as IShow[]);
+
+	const { isLoading } = useSWR(showList, fetcherSecure, {
```

Comment:
useSWR vraća i `data`, nema potrebe da na onSuccess data.shows spremas u state, mozes to koristiti direktno:
```suggestion
	const { isLoading, data } = useSWR(showList, fetcherSecure, {
```